src/cargo/ops/cargo_new.rs

```diff
@@ -17,11 +17,13 @@ use toml;
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub enum VersionControl { Git, Hg, NoVcs }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
I needed Clone to be able to pass changed options (auto-detected `bin`, for example) to `mk`, so duplicated `VersionControl`'s derive line, then removed incompatible `Copy`, thinking that `Debug